src/userAdmin/UserProducts.jsx

```
@@ -1,21 +1,18 @@
-import React, { useState, useEffect, useContext } from "react";
+import React, { useState, useEffect, useContext, useCallback } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)